lib/experimental/new_embed.dart

```diff
@@ -32,12 +32,13 @@ class NewEmbed {
 
   TabController tabController;
   EditorTabView editorTabView;
-  TextAreaElement editorTextArea;
   TestTabView testTabView;
   ConsoleTabView consoleTabView;
 
   ExecutionService executionSvc;
 
+  EditorFactory editorFactory = codeMirrorFactory;
```

Comment:
Does the class need to store this reference, or can it just be used once directly on line 60?

Reply:
I'd like to use the `editorFactory` to drive the test code as well.